src/PanelTraits/AutoSet.php

```diff
@@ -52,7 +53,7 @@ public function getDbColumnTypes()
         foreach ($table_columns as $key => $column) {
             $column_type = $this->model->getConnection()->getSchemaBuilder()->getColumnType($this->model->getTable(), $column);
             $this->db_column_types[$column]['type'] = trim(preg_replace('/\(\d+\)(.*)/i', '', $column_type));
-            $this->db_column_types[$column]['default'] = ''; // no way to do this using DBAL?!
+            $this->db_column_types[$column]['default'] = $this->model->getConnection()->getDoctrineSchemaManager()->listTableDetails($this->model->getTable())->getColumn($column)->getDefault();
```

Comment:
I'd not usually suggest unrelated changes but there seems to be a common connection here:

```
$conn = $this->model->getConnect();
```

This would shorten the above `$column_type` and this monstrosity too.